views/applications/one.html

```diff
@@ -0,0 +1,102 @@
+{% extends "_donut.html" %}
+
+{% block pageHead %}
+  {{ super() }}
+{% endblock %}
+
+{% block header %}
+  {{ super() }}
+{% endblock %}
+
+{% block main %}
+  <div class="introduction">
+    <h1 class="introduction--h1">{{config.actions.edit|capitalize}}</h1>
+    <p class="introduction--p">{{structure.description}}</p>
+  </div>
+  {% include "_basic-form.html" %}
+
+  {% if app['client-id'] %}
```

Comment:
Would like to see this be similar to the endpoints from above; either `app.client` and `app.secret` or `app.client.id` and `app.client.secret`